src/olympia/amo/templates/amo/opensearch.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
-  {# L10n: {0} is an application name like "Firefox" #}
-  <ShortName>{{ _('{0} Add-ons')|format_html(APP.pretty) }}</ShortName>
-  <Url type="text/html"
-       method="get"
-       template="{{ url('search.search')|absolutify }}?q={searchTerms}">
-  </Url>
-  <Image height="16" width="16">data:image/x-icon;base64,AAABAAEAEBAAAAEAIABoBAAAFgAAACgAAAAQAAAAIAAAAAEAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD//wAD2MgAYtzLACwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD//wAC2MgAo9fIAP/YyADy2MsAOwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD//wAB2cgAn9fIAP/XyAD/18gA/9fIAPPZyQA9AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA18gAmtfIAP/XyAD/18gA/9fIAP/XyAD/18kA9NvJADkAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA398ACNjJAPDXyAD/18gA/9fIAP/XyAD/18gA/9jIAPzZyQB/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA//8AAgAAAAD//wAC3s4AH9fJAOzXyAD/18gA/9jJAIoAAAAAAAAAAAAAAAD//wADAAAAAAAAAAD//wAC2MgAntjIANYAAAAAAAAAAAAAAADYyQDL18gA/9fIAP/YyABPAAAAAAAAAADXygBN2MkA79nJAD0AAAAA18gAmdfIAP/YyQD46tUADAAAAADXygBN18gA/9fIAP/XyAD/18kAwebMAAoAAAAA18kAetfIAP/XyADz3MsALNjJAN7XyAD/18gA/9jJANzYyQC318kA+dfIAP/XyAD/18gA/9fIAP/YyADf2MkAuNfJAPrXyAD/18gA/9jIAGLZygA12MkA8NfIAP/XyAD/18gA/9fIAP/XyAD/18gA/9fIAP/XyAD/18gA/9fIAP/XyAD/18gA/9fJAKb//wADAAAAANnLADbYyQDw18gA/9fIAP/XyAD/18gA/9fIAP/XyAD/18gA/9fIAP/XyAD/18gA/9jIAKj//wADAAAAAAAAAAAAAAAA18kAbdfIAP/XyAD/18gA/9fIAP/XyAD/18gA/9fIAP/XyAD/18gA/9fJAOz//wAEAAAAAAAAAADbzAAy2MkAotjJAN3XyAD/18gA/9fIAP/XyAD/18gA/9fIAP/XyAD/18gA/9fIAP/XyAD/2MgArNnIAH4AAAAA18gAp9fIAP/XyAD/18gA4NjKAG/YyQDx18gA/9fIAP/XyAD/18gA/9fJAK7YyQCI18gA/9fIAP/XyAD/2s0AKdfJAG3XyAD/18gA/9jIAKMAAAAA2csANtjJAPDXyAD/18gA/9jJAKX//wAD3MoAHdfIAP/XyAD/2MgA1//MAAUAAAAA2ckAbNfJAKbbzAAyAAAAAAAAAADZygA12MkA3dfIAJr//wADAAAAAAAAAADZyAB+18kAoN3JACYAAAAA/H8AAPg/AADwHwAA8A8AAOAPAADQOwAAHDEAAAgQAAAAAAAAAAAAAIABAADAAwAAAAEAAAAAAAAIAAAAjDEAAA==</Image>
-  <OutputEncoding>UTF-8</OutputEncoding>
-  <InputEncoding>UTF-8</InputEncoding>
-
-</OpenSearchDescription>
```

Comment:
Can we actually remove this?  I thought frontend didn't serve the opensearch file (yet)?

Reply:
You're completely right, I thought https://github.com/mozilla/addons-frontend/issues/3836 had fixed this but it only links to the addons-server opensearch.xml file.

I'll un-delete this then :(